src/amp-url-creator.js

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+// TODO(chenshay) : Write tests once testing capabilites are hooked up for this project.
```

Comment:
Would it make sense to do this first? A change like this could really use a thorough set of unit tests.